lib/Runtime/PlatformAgnostic/Chakra.Runtime.PlatformAgnostic.vcxproj

```diff
@@ -52,12 +52,19 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)Platform\Windows\EventTrace.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)Platform\Windows\PerfTrace.cpp" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IntlICU)'=='true'">
+    <ClCompile Include="$(MSBuildThisFileDirectory)Platform\Common\Intl.cpp" />
+  </ItemGroup>
   <ItemGroup>
     <ClInclude Include="ChakraPlatform.h" />
     <ClInclude Include="EventTrace.h" />
     <ClInclude Include="PerfTrace.h" />
     <ClInclude Include="RuntimePlatformAgnosticPch.h" />
     <ClInclude Include="UnicodeText.h" />
+    <ClInclude Include="IPlatformAgnosticResource.h" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(IntlICU)'=='true'">
+    <ClInclude Include="Intl.h" />
```

Comment:
The naming here is a bit confusing: If IntlICU is true, we include Intl.h, but if IntlWinGlob (I know it doesn't exist but regardless) is true, we don't include Intl.h?

Reply:
Right now `Intl.h` is only needed for `IntlICU` -- in the future we will probably want to move the `WindowsGlobalizationAdapter` into `PlatformAgnostic` and then have i18n-library-specific paths here. The ICU version is not platform-specific but is an implementation detail that can certainly be abstracted over using `PlatformAgnostic` and conditional options (in the same way we choose between platforms in PlatformAgnostic code). For this, I'd probably call the folders `Common` (for WinGlob -- or `Common.WinGlob`) and `Common.ICU` (as we already have `.ICU` as a way to switch over ICU or not ICU in `PlatformAgnostic`).